release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -180,8 +180,8 @@ In {product-title} 4.12, cluster deployments to {rh-openstack-first} clouds that
 For a brief overview of this type of deployment, see the blog post link:https://cloud.redhat.com/blog/deploying-your-cluster-at-the-edge-with-openstack[Deploying Your Cluster at the Edge With OpenStack].
 
 [id="ocp-4-12-aws-outposts"]
-==== {product-title} on AWS Outposts
-{product-title} {product-version} is now supported on the AWS Outposts platform. With AWS Outposts you can deploy edge-based worker nodes, while using AWS Regions for the control plane nodes. The documentation for this feature is currently unavailable and is targeted for release at a later date.
+==== {product-title} on AWS Outposts (Technology Preview)
+{product-title} {product-version} is now supported on the AWS Outposts platform as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. With AWS Outposts you can deploy edge-based worker nodes, while using AWS Regions for the control plane nodes. For more information, see xref:../installing/installing_aws/installing-aws-outposts-remote-workers.adoc[Installing a cluster on AWS with remote workers on AWS Outposts].
```

Comment:
The Installing a cluster on AWS with remove workers on AWS Outposts link does not work.

Reply:
Agree. Link does not work. I suspect this is because the xref hasn't been merged yet? If that's the case, this will need to be marked out AND `xref:../` will need removed because I think without an actual xref, there will be build errors.

Reply:
@eohartman @stevsmit Thanks for taking the time to review. 
So this AWS Outposts content was agreed to be delivered after the OCP 4.12 release. 
For the OCP 4.12 release itself, it only contained the text in line 184 below highlighting that the documentation would be released at a later date. 

After the OCP 4.12 release, the module content 'Installing a cluster on AWS with remove workers on AWS Outposts' was merged. This module is relevant for 4.12+ For more details, see PR here https://github.com/openshift/openshift-docs/pull/55427